index.html

```diff
@@ -2592,9 +2595,13 @@ <h3>Implementation Considerations for ECDSA Algorithms</h3>
 publicized integrity breaches in widely deployed systems. To counter this problem,
 a hash-based method of determining the secret number <em>k</em>, called
 <em>Deterministic ECDSA</em>, is given in [[FIPS-186-5]] and [[RFC6979]].
+        </p>
+        <p>
 Verification of a ECDSA signature is independent of the method of generating
 <em>k</em>. Hence it is generally recommended to use <em>Deterministic
-ECDSA</em> unless other requirements dictate otherwise.
+ECDSA</em> unless other requirements dictate otherwise. For example, using
+different <em>k</em> values results in different signature values for the same
+document which may be a desirable property in some privacy enhancing situations.
```

Comment:
Avoid normative term to avoid generating "testable statement":

```suggestion
document which might be a desirable property in some privacy enhancing situations.
```